src/BottomNavigation/BottomNavigationAction.react.js

```diff
@@ -10,8 +10,10 @@ import Icon from '../Icon';
 const propTypes = {
     /**
     * Will be rendered above the label as a content of the action.
+    * If string, result will be <Icon name={icon} ...rest />
+    * If ReactElement, will be used as is
     */
-    icon: PropTypes.string.isRequired,
+    icon: PropTypes.node.isRequired,
```

Comment:
Could you change it for something like [this](https://github.com/xotahal/react-native-material-ui/blob/master/src/ListItem/ListItem.react.js#L27)?